### PR TITLE
[Bootstrap] Adapt our template for pkgdown website to match their custom UI needs

### PR DESCRIPTION
**Pull request by s-pace**
updating template for pkgdown website to use their UI integration r-lib/pkgdown#639 #397

### CODE REVIEW

File: deployer/src/config_creator.py [hunk 1]
```diff
@@ -74,23 +74,90 @@ def to_gitbook_config(config):
...
+        ))),
+        ("articles", OrderedDict((
+            ("lvl0", ".contents h1"),
+            ("lvl1", ".contents .name"),
+            ("lvl2", OrderedDict((
+                ("selector", ".contents h2, .contents h3"),
+                ("default_value", "Context")
+            ))),
+            ("text",
+             ".contents p, .contents li, .tempate-article .contents .pre")
```
Comment (pixelastic) [inline deployer/src/config_creator.py]: Might be a typo: `tempate-` instead of `.template-`

File: deployer/src/config_creator.py [hunk 2]
```diff
@@ -74,23 +74,90 @@ def to_gitbook_config(config):
...
-    config["custom_settings"] = {"separatorsToIndex": "_"}
-    config["scrap_start_urls"] = False
-    config["stop_urls"] = ["LICENSE-text.html"]
+    config["stop_urls"] = ["/reference/$",
+                           "/reference/index.html",
+                           "/articles/$",
+                           "/articles/index.html"]
+    config["custom_settings"] = {
+        "separatorsToIndex": "_",
+        "attributesToRetrieve": ["hierarchy",
```
Comment (pixelastic) [inline deployer/src/config_creator.py]: Why do we need to overwrite the `attributesToRetrieve`?

File: deployer/src/config_creator.py (hunk 1 above)
Comment (s-pace) [reply deployer/src/config_creator.py]: You are write. I have removed it as the corrected selector from it is not matching any element from the websites.
Fixed in configuration too https://github.com/algolia/docsearch-configs/commit/538370ba764bd45102258c00644f2a86fee49588

File: deployer/src/config_creator.py (hunk 2 above)
Comment (s-pace) [reply deployer/src/config_creator.py]: It comes from the need from pkgdown that uses a different UI, they process record in another way:

image

**Pull request merged**